runtime/compiler/p/codegen/J9TreeEvaluator.cpp

```diff
@@ -1070,10 +1070,11 @@ void generateReportFieldAccessOutlinedInstructions(TR::Node *node, TR::LabelSymb
    auto linkageProperties = linkage->getProperties();
    TR::Register *valueReferenceReg = NULL;
    TR::Register *jumpReg = cg->allocateRegister();
+   TR::Register *decompressedValueReg = cg->allocateRegister();
 
```

Comment:
you are allocating a new register no matter if it will be used.  you probably need to give some more thoughts whether it can be done better. for example, whether an existing register can be re-used as long as no overlapping scopes of use.

Reply:
It looks like that `valueReg` isn't used anymore outside of `generateReportFieldAccessOutlinedInstructions`. So instead of using a new register for the decompression, I just clobbered the existing value in `valueReg`.